modules/api/src/main/java/org/apache/ignite/lang/ErrorGroup.java

```diff
@@ -17,46 +17,49 @@
 
 package org.apache.ignite.lang;
 
-import static java.util.regex.Pattern.DOTALL;
 import static org.apache.ignite.lang.ErrorGroups.errorGroupByCode;
 
 import java.util.HashSet;
 import java.util.Set;
 import java.util.UUID;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-import org.jetbrains.annotations.Nullable;
 
 /**
  * This class represents a concept of error group. Error group defines a collection of errors that belong to a single semantic component.
  * Each group can be identified by a name and an integer number that both must be unique across all error groups.
  */
 public class ErrorGroup {
-    /** Additional prefix that is used in a human-readable format of ignite errors. */
-    public static final String ERR_PREFIX = "IGN-";
-
-    /** Error message pattern. */
-    private static final Pattern EXCEPTION_MESSAGE_PATTERN =
-            Pattern.compile("(.*)(IGN)-([A-Z]+)-(\\d+)\\s(TraceId:)([a-f0-9]{8}(?:-[a-f0-9]{4}){4}[a-f0-9]{8})(\\s?)(.*)", DOTALL);
-
     /** Group name. */
     private final String groupName;
 
     /** Group code. */
     private final short groupCode;
 
+    /** Additional prefix that is used in a human-readable format of error messages. */
+    private final String errorPrefix;
```

Comment:
Hmm, I'm a little confused by the name. Perhaps, it would be nice to change to `groupNamePrefix`, or `vendorSpecificPrefix` etc. What do you think?

Reply:
If we honestly it's not so important and all variants reflect nature of the field but from different perspective